### ISSUE TITLE
iter_batched comments should say clone not to_vec

### ISSUE THREAD

**Issue by jkugelman** [bheisler/criterion.rs#715]
In `bencher.rs` there are four doc comments like this:

```rust
/// c.bench_function("with_setup", move |b| {
///     // This will avoid timing the to_vec call.
///     b.iter_batched(|| data.clone(), /* ... */)
/// });
```

They should say `clone` not `to_vec` because the setup code is `|| data.clone()`.

**Comment by Marcono1234** [comment]
Should this be closed? It looks like it was fixed by #716.

**Comment by waywardmonkeys** [comment]
Yes. @lemmih, please close this.
